packages/gatsby/src/pages/index.js

```diff
@@ -191,27 +196,27 @@ const IndexPage = ({data, searchState, onSearchStateChange}) => {
           <Section>
             <SectionContent>
               <SellingPoints>
-                <SellingPoint imgUrl={agendaIcon}>
+                <SellingPoint imgUrl={agendaIcon} url="/features/workspaces">
                   <h3>Workspaces</h3>
                   Split your project into sub-components kept within a single repository.
                 </SellingPoint>
-                <SellingPoint imgUrl={laptopIcon}>
+                <SellingPoint imgUrl={laptopIcon} url="/configuration/yarnrc#yarnUpgradePath">
```

Comment:
Imo we should add an entry into Q&A titled "How is Yarn 'deterministic'?" and link it from here.

Reply:
Do you want to wait until this article is written or should we merge this one first and then change the link when the article is created?